src/script/lua_api/l_server.cpp

```diff
@@ -45,6 +45,17 @@ int ModApiServer::l_get_server_status(lua_State *L)
 	return 1;
 }
 
+// print(text)
+int ModApiServer::l_print(lua_State *L)
```

Comment:
Two issues:

1. This function should be in l_util.cpp. Otherwise you can't use print() in any Lua environment that isn't in-game.
2. For compatibility with vanilla Lua and previous minetest versions, print() should accept a variable number of arguments, call tostring() on each of them, and write them separated by '\t' and followed by a newline. Like in the [old C++](https://github.com/minetest/minetest/blob/0.4.13/src/script/lua_api/l_util.cpp#L38) or the [new Lua](https://github.com/minetest/minetest/blob/b6dfae02215d7bd61e06ff1999ee3ecb0ec80b9e/builtin/init.lua#L9) definition of core.debug().